Helper/Data.php

```diff
@@ -58,10 +62,20 @@ public function getConversion($product)
         $productUnit = $product->getData('baseprice_product_unit');
         $referenceUnit = $product->getData('baseprice_reference_unit');
 
-        $configArray = unserialize($this->scopeConfig->getValue(
+        $scopeConfig = $this->scopeConfig->getValue(
             self::CONVERSION_CONFIG_PATH,
             \Magento\Store\Model\ScopeInterface::SCOPE_STORE
-        ));
+        );
+
+        $objectManager = ObjectManager::getInstance();
```

Comment:
Please inject necessary classes via di.xml and constructor instead of the object manager :-)